maintainers/maintainer-list.nix

```diff
@@ -7875,6 +7875,12 @@
     githubId = 6445082;
     name = "Joseph Lukasik";
   };
+  jgs = {
```

Comment:
Nitpick: it's quite handy when you maintainer handle is the same as you github handle. ofborg can go though the indirection but it's tedious for humans.